create.go

```diff
@@ -48,6 +49,9 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 		},
 	},
 	Action: func(context *cli.Context) error {
+		if !context.Args().Present() || context.NArg() > 1 {
```

Comment:
yes, you are right. we can just use `if context.NArg() != 1`, I will update the patch.